packages/core/test/model.test.ts

```diff
@@ -488,4 +488,26 @@ describe('createModel', () => {
     // @ts-expect-error
     model.actions.test();
   });
+
+  it('should allow for the action type to be explicitly given when creators have not been configured', () => {
```

Comment:
@Andarist I don't think we should support this; one of the biggest purposes of `createModel` is to _avoid_ specifying generic types. (Also, the tests fail)

Reply:
I don't think this is super useful either, I've just added this since we already support the same for events:
https://github.com/statelyai/xstate/blob/095e40652c3e83b440d1c0b546df64bafc263dd0/packages/core/test/model.test.ts#L28-L31

As to the tests - silly mistake. I forgot that our type tests are also executed at runtime and didn't account for that. I've already fixed that.